### PR TITLE
Add the missing implementation of IdnMapping.GetAscii

### PR DESCRIPTION
**Pull request by tarekgh**
we should have this implementation in place but no idea why got removed in the first place

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @jkotas @weshaggard @danmosemsft

File: src/mscorlib/src/System/Globalization/IdnMapping.cs [hunk 1]
```diff
@@ -130,9 +130,64 @@ public String GetAscii(String unicode, int index)
             return GetAscii(unicode, index, unicode.Length - index);
         }
 
-        public String GetAscii(String unicode, int index, int count)
+        public string GetAscii(String unicode, int index, int count)
         {
-            throw null;
+            if (unicode == null) throw new ArgumentNullException("unicode");
```
Comment (danmosemsft) [inline src/mscorlib/src/System/Globalization/IdnMapping.cs]: nameof(unicode)

Comment (danmosemsft) [inline src/mscorlib/src/System/Globalization/IdnMapping.cs]: and other parameters below

**Comment by tarekgh** [comment]
@danmosemsft 

> This is implemented in corefx\src\Common\src\System\Globalization\IdnMapping.cs. I'm not sure why it's there and in mscorlib.

the work I am doing now is for getting rid of the implementation in corefx and have unified implementation in coreclr that all other libraries can use

**Comment by danmosemsft** [comment]
Ah, makes sense.

**Pull request merged**